torch/optim/swa_utils.py

```diff
@@ -100,8 +100,10 @@ def avg_fn(averaged_model_parameter, model_parameter, num_averaged):
     def forward(self, *args, **kwargs):
         return self.module(*args, **kwargs)
 
-    def update_parameters(self, model):
-        for p_swa, p_model in zip(self.parameters(), model.parameters()):
+    def update_parameters(self, model, use_state_dict=False):
```

Comment:
I seems more natural to define this on the constructor as one won't switch between different options in the middle of the updates.

Reply:
Makes sense.